util.sh

```diff
@@ -1,8 +1,8 @@
-#!/bin/bash
+# -*- indent-tabs-mode: nil  -*-
 #********************************************************************
 # Software License Agreement (BSD License)
 #
-#  Copyright (c) 2016, University of Colorado, Boulder
+#  Copyright (c) 2018, Bielefeld University
```

Comment:
this is odd to me you're changing the copyright but i guess you've changed this file enough for it to be a different file. i wouldn't make this a habit but i don't really care here.

Reply:
This file is completely new. The copyright is not on the file, but it's content, right.
Note, that your still mentioned as a co-author as you developed some original ideas for it.

> ... I don't really care here.

I wouldn't care though as well.